src/Compilers/VisualBasic/Test/Semantic/BasicCompilerSemanticTest.vbproj

```diff
@@ -220,5 +91,21 @@
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="Semantics\BinaryOperatorsTestSource1.vb" />
```

Comment:
I see now that this is a similar situation where the file extensions are all .vb. You can disregard this.